homeassistant/components/tuya/light.py

```diff
@@ -100,6 +100,7 @@ class TuyaLightEntityDescription(LightEntityDescription):
     "dj": (
         TuyaLightEntityDescription(
             key=DPCode.SWITCH_LED,
+            name="Light",
```

Comment:
This change is incorrect, it should inherit the device name as this is the primary entity. Please revert this change.